doc/install_mne_c.rst

```diff
@@ -2,290 +2,175 @@
+To configure MNE-C automatically for ``zsh``, use the built-in ``emulate``
+command in your ``.zshrc`` file:
 
-.. code-block:: console
+.. code-block:: sh
 
-    $ export MNE_ROOT=<MNE>
-    $ export MATLAB_ROOT=<Matlab>
-    $ emulate sh -c 'source $MNE_ROOT/bin/mne_setup_sh'
+    export MNE_ROOT=<path_to_MNE>
+    export MATLAB_ROOT=<path_to_MATLAB>
+    emulate sh -c 'source $MNE_ROOT/bin/mne_setup_sh'
 
-For ``csh`` or ``tcsh`` shell, the corresponding commands are:
+To configure MNE-C automatically for ``csh`` or ``tcsh`` shells, the
+corresponding commands in the ``.cshrc`` / ``.tcshrc`` file are:
 
-.. code-block:: console
+.. code-block:: sh
 
-    $ setenv MNE_ROOT <MNE>
-    $ setenv MATLAB_ROOT <Matlab>
-    $ source $MNE_ROOT/bin/mne_setup
-
-
-For BEM mesh generation using the watershed algorithm or
-on the basis of multi-echo FLASH MRI data (see :ref:`create_bem_model`) and
-for accessing the tkmedit program
-from mne_analyze, see :ref:`CACCHCBF`,
-the MNE software needs access to a FreeSurfer license
-and software. Therefore, to use these features it is mandatory that
-you set up the FreeSurfer environment
-as described in the FreeSurfer documentation.
-
-The environment variables relevant to the MNE software are
-listed in :ref:`CIHDGFAA`.
-
-.. tabularcolumns:: |p{0.3\linewidth}|p{0.55\linewidth}|
-.. _CIHDGFAA:
-.. table:: Environment variables
-
-    +-------------------------+--------------------------------------------+
-    | Name of the variable    |   Description                              |
-    +=========================+============================================+
-    | ``MNE_ROOT``            | Location of the MNE software, see above.   |
-    +-------------------------+--------------------------------------------+
-    | ``FREESURFER_HOME``     | Location of the FreeSurfer software.       |
-    |                         | Needed during FreeSurfer reconstruction    |
-    |                         | and if the FreeSurfer MRI viewer is used   |
-    |                         | with mne_analyze, see :ref:`CACCHCBF`.     |
-    +-------------------------+--------------------------------------------+
-    | ``SUBJECTS_DIR``        | Location of the MRI data.                  |
-    +-------------------------+--------------------------------------------+
-    | ``SUBJECT``             | Name of the current subject.               |
-    +-------------------------+--------------------------------------------+
-    | ``MNE_TRIGGER_CH_NAME`` | Name of the trigger channel in raw data,   |
-    |                         | see :ref:`mne_process_raw`.                |
-    +-------------------------+--------------------------------------------+
-    | ``MNE_TRIGGER_CH_MASK`` | Mask to be applied to the trigger channel  |
-    |                         | values, see :ref:`mne_process_raw`.        |
-    +-------------------------+--------------------------------------------+
```

Comment:
MNE_TRIGGER_CH_MASK and MNE_TRIGGER_CH_NAME variables are not documented anymore?

Reply:
fixed in 897cbbb